# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26/config_h-compat.patch
# Copyright (C) 2006 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-2.6.19-dist/include/linux/config.h.orig       1970-01-01 01:00:00.000000000 +0100
+++ linux-2.6.19-dist/include/linux/config.h    2006-12-05 22:18:31.000000000 +0100
@@ -0,0 +1,4 @@
+#ifndef _LINUX_CONFIG_H
+#define _LINUX_CONFIG_H
+#include <linux/autoconf.h>
+#endif
